app/javascript/components/close_contact/CloseContact.js

```diff
@@ -132,137 +142,151 @@ class CloseContact extends React.Component {
+                onChange={this.handleChange}
+                value={this.state.assigned_user || ''}
+              />
+              <datalist id="assigned_users">
+                {this.props.assigned_users?.map(num => {
+                  return (
+                    <option value={num} key={num}>
+                      {num}
+                    </option>
+                  );
+                })}
+              </datalist>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['assigned_user']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
+          <Row>
+            <Form.Group as={Col}>
+              <Form.Label htmlFor="notes" className="nav-input-label">
+                Notes
+                {schema?.fields?.notes?._exclusive?.required && '*'}
+              </Form.Label>
+              <Form.Control
+                id="notes"
+                as="textarea"
+                rows="5"
+                className="form-square"
+                value={this.state.notes || ''}
+                placeholder={this.closeContactNotePlaceholder}
+                maxLength="2000"
+                onChange={this.handleChange}
+              />
+              <Form.Label className="notes-character-limit"> {2000 - this.state.notes.length} characters remaining </Form.Label>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['notes']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
         </Modal.Body>
         <Modal.Footer>
           <Button variant="secondary btn-square" onClick={toggle}>
             Cancel
           </Button>
-          <Button
-            variant="primary btn-square"
-            onClick={submit}
-            disabled={
-              !(this.state.first_name || this.state.last_name || this.state.email || this.state.primary_telephone || this.state.notes) || this.state.loading
-            }>
-            {this.props.close_contact.id ? 'Update' : 'Create'}
+          <Button variant="primary btn-square" onClick={submit} disabled={this.state.disableCreate || this.state.loading}>
+            <span data-for="create-tooltip" data-tip="" className="ml-1">
+              {this.props.close_contact.id ? 'Update' : 'Create'}
+            </span>
           </Button>
+          {/* Typically we pair the ReactTooltip up directly next to the mount point. However, due to the disabled attribute on the button */}
+          {/* above, this Tooltip should be placed outside the parent component (to prevent unwanted parent opacity settings from being inherited) */}
+          {/* This does not impact component functionality at all. */}
+          <ReactTooltip
+            id="create-tooltip"
+            multiline={true}
+            place="top"
+            type="dark"
+            effect="solid"
+            offset={{ left: 113 }}
```

Comment:
I don't think we need this offset, tooltips on other modals hang off the side and I don't love hardcoding this anyway

Reply:
Not sure how much I agree with this, but I can take a look.

Reply:
I don't love it either but it also makes it clearer what the tooltip is attached too.  If we could have the arrow back though I'd be more okay with this